locale/en-us.yaml

```diff
@@ -1708,3 +1708,7 @@ solver_err:
   other: The Platform failed to resolve the dependencies for this build. {{.V0}}\n{{.V1}}
 transient_solver_tip:
   other: You may want to retry this command if the failure was related to a resourcing or networking issue.
+warning_git_project_mismatch:
+  other: |
+    [ERROR]Warning:[/RESET] The activestate.yaml used on "[NOTICE]<repo-url>[/RESET]" does not match your project; "[NOTICE]<project namespace>[/RESET]".
+    We're updating it to match your current project. For more information check out <documentation-url>.
```

Comment:
You didn't replace the tags. Things like `<project namespace>` should be replaced with the actual values.

Reply:
Whoops, that's a copy-paste fail 🤦‍♂️ .

I opted to add the repo URL as an argument to the `ensureCorrectProject` function rather than do any restructuring as I thought this would be faster.